.gitignore

```diff
@@ -25,3 +25,108 @@ docs/api/*.txt
 .mypy_cache/
 .eggs
 dulwich.dist-info
+
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+# C extensions
+*.so
+
+# Distribution / packaging
+.Python
+build/
+develop-eggs/
+dist/
+downloads/
+eggs/
+.eggs/
+lib/
+lib64/
+parts/
+sdist/
+var/
+wheels/
+*.egg-info/
+.installed.cfg
+*.egg
+MANIFEST
+
+# PyInstaller
+#  Usually these files are written by a python script from a template
+#  before PyInstaller builds the exe, so as to inject date/other infos into it.
+*.manifest
+*.spec
+
+# Installer logs
+pip-log.txt
+pip-delete-this-directory.txt
+
+# Unit test / coverage reports
+htmlcov/
+.tox/
+.coverage
+.coverage.*
+.cache
+nosetests.xml
+coverage.xml
+*.cover
+.hypothesis/
+.pytest_cache/
+
+# Translations
```

Comment:
These are not relevant.

Reply:
In fact, these are the ignore lists that are often encountered in Python development. For example, `.mo` and `.pot` can provide localized languages.

Reply:
yes, but for dulwich it's not relevant. And when it becomes relevant we can add these ignores.